api/src/main/java/demo/controller/UserController.java

```diff
@@ -23,52 +21,54 @@
 import static org.springframework.http.ResponseEntity.created;
 
 @Slf4j
+@Valid
```

Comment:
Maybe I'm mistaken but the `@Valid` annotation is method attribute level annotation, not class level

Reply:
You can add it at a class level and it will apply to all functions in said class